parsers/US_MISO.py

```diff
@@ -113,6 +118,45 @@ def fetch_production(zone_key='US-MISO', session=None, target_datetime=None, log
     return data
 
 
+def fetch_wind_forecast(zone_key='US-MISO', session=None, target_datetime=None, logger=None):
+    """
+    Requests the day ahead wind forecast (in MW) of a given zone
+    Arguments:
+    zone_key (optional) -- used in case a parser is able to fetch multiple countries
+    session (optional)  -- request session passed in order to re-use an existing session
+    target_datetime (optional) -- used if parser can fetch data for a specific day
+    logger (optional) -- handles logging when parser is run as main
+    Return:
+    A list of dictionaries in the form:
+    {
+    'source': 'misoenergy.org',
+    'value': 12932.0,
+    'datetime': '2019-01-01T00:00:00Z',
+    'zoneKey': 'US-MISO'
+    }
+    """
+
+    s = session or requests.Session()
+    req = s.get(wind_forecast_url)
+    raw_json = req.json()
+    raw_data = raw_json['Forecast']
+
+    data = []
+    for item in raw_data:
+        dt = parser.parse(item['DateTimeEST']).replace(tzinfo=tz.gettz('America/New_York'))
+        value = float(item['Value'])
+
+        datapoint = {'datetime': dt,
+                     'value': value,
```

Comment:
instead of `value`: .. you should return `production: {wind: value}` as multiple forecasts can be returned
```suggestion
                     'production': {'wind': value},
```